recipes/kartothek/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "kartothek" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 59b872febe0acee7fbead8b07f012a0ab631d760aa8f03e8650d2841765fac19
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - setuptools_scm
+    - python >=3.6
+  run:
+    - dask
+    - decorator
+    - funcsigs
+    - msgpack-python >=0.5.2
+    - numpy !=1.15.0,!=1.16.0
```

Comment:
Is this really needed? It looks like it is in `requirements.txt` because of a dask conflict in PyPI, but seems unnecessary for conda. (note that current jobs are using `1.16.3`)

Reply:
or you can let is just as ``numpy >1.16``

Reply:
There were real issues with these numpy versions that were fixed in the following patch releases each. I would like to keep the requirement here as the package also works with `numpy==1.15.1`.